Annoucements/_posts/Legal-Activism-DAO/2021-08-03-Legal-Activism-DAO.md

```diff
@@ -1,17 +1,17 @@
 ---
 layout: post
-title:  "Proposal Fund Builder First Legal Activism DAO"
+title:  "Proposta para Obtenção de Fundos do Primeiro DAO de Ativismo Legal"
 categories: [ Annoucements ]
 image: ./Legal-Activism-DAO/1.jpg
 author: LexPunk DAO
-translator:
+translator: jameskbh
 publish: false
```

Comment:
need to change `publish: true` when you translate something

Reply:
Changed